src/commands/emulators-exec.ts

```diff
@@ -16,7 +16,7 @@ import { FirestoreEmulator } from "../emulator/firestoreEmulator";
 async function runScript(script: string): Promise<void> {
   utils.logBullet(`Running script: ${clc.bold(script)}`);
 
-  const env: NodeJS.ProcessEnv = {};
```

Comment:
@samtstern was there any reason for blocking `env` int the first place?

Reply:
No I thought it would be additive.  I just wanted to add one var to the env.

Reply:
ack